docs/pages/guides/basic-layouts.md

```diff
@@ -54,7 +54,7 @@ Update output to render a [`View`](../api/components/View.md) component with an
           type: :view,
           background_color_rgba: "#4d4d4dff",
           children: [
-            %{ type: :input_stream, input_id: :input_1 }
```

Comment:
I decided to use strings instead of atoms for input ids everywhere for consistency.
It's also the typing used in docs: https://hexdocs.pm/membrane_live_compositor_plugin/Membrane.LiveCompositor.html#t:input_id/0